### DISCUSSION TITLE
Run on Boot

### DISCUSSION THREAD

**Discussion by blackwellj** [chuot/rdio-scanner#146] [Q&A] [answered]
What is the best way to run on boot on a pi?

**Comment by chuot** [comment, marked as answer]
It would be the same as all other platforms. Use the built-in service manager with the `-service` argument. Refer to page 6 of the PDF document provided with Rdio Scanner for examples. Be aware though that folder name with spaces, like `Rdio Scanner` for example, doesn't seem to work on RPI because the underlying library has a bug with spaces.
